docs/configuration/index.md

```diff
@@ -800,6 +800,7 @@ Prior to version 0.13.0, Druid string columns treated `''` and `null` values as
 |Property|Description|Default|
 |---|---|---|
 |`druid.generic.useDefaultValueForNull`|Set to `false` to store and query data in SQL compatible mode. When set to `true` (legacy mode), `null` values will be stored as `''` for string columns and `0` for numeric columns.|`false`|
+|`druid.generic.useThreeValueLogic`|Set to `true` to use SQL compatible three-value logic when processing Druid filters when `druid.generic.useDefaultValueForNull` is `false` and `druid.expressions.useStrictBooleans` is `true`. When set to `false` Druid uses 2 value logic for filter processing, even when `druid.generic.useDefaultValueForNull=false` and `druid.expressions.useStrictBooleans` is true.|`true`|
```

Comment:
Shouldn't this be `druid.generic.useThreeValueLogicForNativeFilters`? A mouthful, but isn't 3VL still used inside expressions when strict booleans is true and this is false?

Also, nit: consistency between "`druid.generic.useDefaultValueForNull=false`" vs "`druid.expressions.useStrictBooleans` is true"

Also, suggest linking to `../querying/sql-data-types.md` for more info.

Reply:
changed